src/components/Checkbox.js

```diff
@@ -13,27 +13,35 @@ const propTypes = {
     onPress: PropTypes.func.isRequired,
 
     /** Should the input be styled for errors  */
```

Comment:
Looks like this comment is out-of-date

Reply:
Earlier we were applying the error styles when `hasError` was true, now we are just checking if errorText is present, so the functionality hasn't change and so I think we can keep this comment.

Reply:
@thesahindia "Should the input be styled for errors" gives the impression that it's a boolean.

```suggestion
    /** Error text to display */
```

Reply:
@rushatgabhane, Yes makes sense, but now I think a better way would be to keep it as hasError since we are only going to use it as boolean and I will pass it like this -
 ```js
               <Checkbox
                    hasError={!!props.errorText}
```
the `errorText` that we get in `CheckboxWithLabel` is to show the error message and for checkbox component we only need a boolean value

Reply:
@thesahindia good catch, agreed!

Reply:
@thesahindia good call. I agree, let's do that.

`hasError={Boolean(props.errorText)}`

Reply:
Done ✔️

Reply:
@tgolen all yours! LGTM